charts/tests/helmunit_test.go

```diff
@@ -0,0 +1,118 @@
+//go:build helmunit
+
+package test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/gkampitakis/go-snaps/snaps"
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+)
+
+func TestMain(m *testing.M) {
+	code := m.Run()
+
+	// After all tests have run `go-snaps` will sort snapshots
+	snaps.Clean(m, snaps.CleanOpts{Sort: true})
+
+	os.Exit(code)
+}
+
+// An example of how to verify the rendered template object of a Helm Chart given various inputs.
+func TestHelmNICTemplate(t *testing.T) {
+	t.Parallel()
+
+	tests := map[string]struct {
+		valuesFile  string
+		releaseName string
+		namespace   string
+		expectedErr error
```

Comment:
is the `expectedErr` field used in the tests?

Reply:
No, I will remove it now